### PR TITLE
Adding Identity integration and PO-UI

### PR DESCRIPTION
**Pull request by felabs01**

### CODE REVIEW

File: {{cookiecutter.project_slug}}/README.md [hunk 1]
```diff
@@ -1,14 +1,31 @@
-# {{cookiecutter.project_name}}
+# SeedCarolApp
```
Comment (gzamboni) [inline {{cookiecutter.project_slug}}/README.md:1]:
```suggestion
# {{cookiecutter.project_name}}
```
IMHO I think we can change the project name here.. not fixed to SeedCarolApp

File: {{cookiecutter.project_slug}}/README.md [hunk 2]
```diff
@@ -1,14 +1,31 @@
...
 
-{{cookiecutter.project_short_description}}
+This project can be used as a seed when you want to start building a Carol app.
```
Comment (gzamboni) [inline {{cookiecutter.project_slug}}/README.md:3]:
```suggestion
{{cookiecutter.project_short_description}}

# About Project template
This project can be used as a seed when you want to start building a Carol app. 
```

File: {{cookiecutter.project_slug}}/angular.json [hunk 3]
```diff
@@ -3,21 +3,22 @@
   "version": 1,
   "newProjectRoot": "projects",
   "projects": {
-    "{{cookiecutter.project_slug}}": {
+    "seed-carol-app": {
```
Comment (gzamboni) [inline {{cookiecutter.project_slug}}/angular.json:6]:
```suggestion
    "{{cookiecutter.project_slug}}": {
```

File: {{cookiecutter.project_slug}}/angular.json [hunk 4]
```diff
@@ -3,21 +3,22 @@
...
+          "style": "scss"
         }
       },
       "architect": {
         "build": {
           "builder": "@angular-devkit/build-angular:browser",
           "options": {
-            "outputPath": "dist/{{cookiecutter.project_slug}}",
+            "aot": true,
+            "outputPath": "dist/seed-carol-app",
```
Comment (gzamboni) [inline {{cookiecutter.project_slug}}/angular.json:21]:
```suggestion
            "outputPath": "dist/{{cookiecutter.project_slug}}",
```

File: {{cookiecutter.project_slug}}/angular.json [hunk 5]
```diff
@@ -64,19 +68,19 @@
         "serve": {
           "builder": "@angular-devkit/build-angular:dev-server",
           "options": {
-            "browserTarget": "{{cookiecutter.project_slug}}:build",
+            "browserTarget": "seed-carol-app:build",
```
Comment (gzamboni) [inline {{cookiecutter.project_slug}}/angular.json:71]:
```suggestion
            "browserTarget": "{{cookiecutter.project_slug}}:build",
```

File: {{cookiecutter.project_slug}}/angular.json [hunk 6]
```diff
@@ -64,19 +68,19 @@
...
             "proxyConfig": "proxy.conf.json"
           },
           "configurations": {
             "production": {
-              "browserTarget": "{{cookiecutter.project_slug}}:build:production"
+              "browserTarget": "seed-carol-app:build:production"
```
Comment (gzamboni) [inline {{cookiecutter.project_slug}}/angular.json:76]:
```suggestion
              "browserTarget": "{{cookiecutter.project_slug}}:build:production"
```

File: {{cookiecutter.project_slug}}/angular.json [hunk 7]
```diff
@@ -64,19 +68,19 @@
...
             }
           }
         },
         "extract-i18n": {
           "builder": "@angular-devkit/build-angular:extract-i18n",
           "options": {
-            "browserTarget": "{{cookiecutter.project_slug}}:build"
+            "browserTarget": "seed-carol-app:build"
```
Comment (gzamboni) [inline {{cookiecutter.project_slug}}/angular.json:83]:
```suggestion
            "browserTarget": "{{cookiecutter.project_slug}}:build"
```

File: {{cookiecutter.project_slug}}/angular.json [hunk 8]
```diff
@@ -110,7 +115,7 @@
         }
       }
     },
-    "{{cookiecutter.project_slug}}-e2e": {
+    "seed-carol-app-e2e": {
```
Comment (gzamboni) [inline {{cookiecutter.project_slug}}/angular.json:118]:
```suggestion
    "{{cookiecutter.project_slug}}-e2e": {
```

File: {{cookiecutter.project_slug}}/angular.json [hunk 9]
```diff
@@ -139,5 +144,5 @@
       }
     }
   },
-  "defaultProject": "{{cookiecutter.project_slug}}"
+  "defaultProject": "seed-carol-app"
```
Comment (gzamboni) [inline {{cookiecutter.project_slug}}/angular.json:147]:
```suggestion
  "defaultProject": "{{cookiecutter.project_slug}}"
```

File: {{cookiecutter.project_slug}}/angular.json [hunk 10]
```diff
@@ -119,11 +124,11 @@
           "builder": "@angular-devkit/build-angular:protractor",
           "options": {
             "protractorConfig": "e2e/protractor.conf.js",
-            "devServerTarget": "{{cookiecutter.project_slug}}:serve"
+            "devServerTarget": "seed-carol-app:serve"
           },
           "configurations": {
             "production": {
-              "devServerTarget": "{{cookiecutter.project_slug}}:serve:production"
+              "devServerTarget": "seed-carol-app:serve:production"
```
Comment (gzamboni) [inline {{cookiecutter.project_slug}}/angular.json:131]:
```suggestion
              "devServerTarget": "{{cookiecutter.project_slug}}:serve:production"
```

File: {{cookiecutter.project_slug}}/angular.json [hunk 11]
```diff
@@ -119,11 +124,11 @@
           "builder": "@angular-devkit/build-angular:protractor",
           "options": {
             "protractorConfig": "e2e/protractor.conf.js",
-            "devServerTarget": "{{cookiecutter.project_slug}}:serve"
+            "devServerTarget": "seed-carol-app:serve"
```
Comment (gzamboni) [inline {{cookiecutter.project_slug}}/angular.json:127]:
```suggestion
            "devServerTarget": "{{cookiecutter.project_slug}}:serve"
```

File: {{cookiecutter.project_slug}}/package.json [hunk 12]
```diff
@@ -1,61 +1,52 @@
 {
-  "name": "{{cookiecutter.project_slug}}",
-  "version": "{{cookiecutter.version}}",
-  "author": {
-      "name": "{{ cookiecutter.user_full_name }}",
-      "email": "{{ cookiecutter.user_email }}",
-      "url": "https://github.com/{{cookiecutter.github_username}}"
-  },
+  "name": "seed-carol-app",
+  "version": "0.0.1",
```
Comment (gzamboni) [inline {{cookiecutter.project_slug}}/package.json:3]:
```suggestion
  "name": "{{cookiecutter.project_slug}}",
  "version": "{{cookiecutter.version}}",
  "author": {
      "name": "{{ cookiecutter.user_full_name }}",
      "email": "{{ cookiecutter.user_email }}",
      "url": "https://github.com/{{cookiecutter.github_username}}"
  },
```

File: {{cookiecutter.project_slug}}/proxy.conf.json [hunk 13]
```diff
@@ -1,9 +1,9 @@
 {
   "connectorId": "0a0829172fc2433c9aa26460c31b78f0",
-  "organization": "{{ cookiecutter.carol_app_organization }}",
-  "environment": "{{ cookiecutter.carol_app_environment }}",
+  "organization": "totvsdevelop",
+  "environment": "totvslabsdevelop",
```
Comment (gzamboni) [inline {{cookiecutter.project_slug}}/proxy.conf.json:4]:
```suggestion
  "organization": "{{ cookiecutter.carol_app_organization }}",
  "environment": "{{ cookiecutter.carol_app_environment }}",
```

File: {{cookiecutter.project_slug}}/proxy.conf.json [hunk 14]
```diff
@@ -1,9 +1,9 @@
...
   "/api/*": {
-      "target": "https://{{ cookiecutter.carol_app_organization }}.carol.ai",
+      "target": "https://totvsdevelop.qarol.ai",
```
Comment (gzamboni) [inline {{cookiecutter.project_slug}}/proxy.conf.json:6]:
```suggestion
      "target": "https://{{ cookiecutter.carol_app_organization }}.qarol.ai",
```

**Review by gzamboni — CHANGES_REQUESTED** [review]
Some cookiecutters tags removed, this remove all automatic configuration and customization.

**Comment by felabs01** [comment]
I am creating a new branch with the latest seed.

**Pull request closed without merging**